wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientTableViewCell.swift

```diff
@@ -159,18 +132,21 @@ extension ClientTableViewCellModel {
 
     private typealias DeviceDetailsSection = L10n.Localizable.Device.Details.Section
 
-    static func from(userClient: UserClient) -> ClientTableViewCellModel {
-        let title = userClient.isLegalHoldDevice ?  L10n.Localizable.Device.Class.legalhold : (userClient.model ?? "")
-
+    static func from(userClient: UserClientType, shouldSetType: Bool = true) -> ClientTableViewCellModel {
+        var title = ""
+        if shouldSetType {
+            title = userClient.deviceClass == .legalHold ?  L10n.Localizable.Device.Class.legalhold : (userClient.deviceClass?.localizedDescription.capitalized ?? userClient.type.localizedDescription.capitalized)
```

Comment:
It's not clear from the code why not to use `userClient.type.localizedDescription.capitalized` every time. Why do we need to check if `userClient.deviceClass` is nil. I would create a helper method or property

Reply:
This was the way it was done. I am merely applying the same logic as before